x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java

```diff
@@ -142,6 +142,7 @@
 public class CsvTests extends ESTestCase {
 
     private static final Logger LOGGER = LogManager.getLogger(CsvTests.class);
+    private static final String IGNORED_CSV_FILE_NAMES_PATTERN = "-IT_tests_only";
```

Comment:
Nothing to do with the fix in this PR, but I believe these field names are more clear like this. `-ignoreCsvTests` were suggesting imo that the whole file is ignored in all test suites. In fact, those fields are ignored in CSV unit tests only.